goreleaser.yaml

```diff
@@ -67,5 +67,8 @@ changelog:
 release:
   prerelease: "auto"
   footer: |
+    ## Release Kustomize Manifests
+    Release manifests can be found at `oci://ghcr.io/authzed/spicedb-operator-manifests:v{{ .Version }}` and can be installed or inspected with [kustomizer](https://kustomizer.dev/)
```

Comment:
I think we need more documentation here for folks to really understand how to use the manifests.